### PR TITLE
Adding new chains

### PR DESCRIPTION
**Pull request by leohar**
- added Kico, Litenry, Crab
- replaced Composable icon in Crowdloans

### CODE REVIEW

File: chains/v2/chains_dev.json [hunk 1]
```diff
@@ -1950,5 +1950,87 @@
         "color": "linear-gradient(315deg, #132849 0%, #2C5AA6 100%)",
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Zeitgeist.svg",
         "addressPrefix": 73
+    },
+    {
+        "chainId": "0x52149c30c1eb11460dce6c08b73df8d53bb93b4a15d0a2e7fd5dafe86a73c0da",
```
[the remark this answers is not in the document]
Comment (leohar) [reply chains/v2/chains_dev.json]: fixed

**Comment by stepanLav** [comment]
## KICO ✅
<details>
  <summary>KICO</summary>
  
Screenshot 2022-02-25 at 07 17 35
Screenshot 2022-02-25 at 07 18 04
Screenshot 2022-02-25 at 07 18 24
Screenshot 2022-02-25 at 07 19 07
Screenshot 2022-02-25 at 07 19 30
Screenshot 2022-02-25 at 07 20 21

</details>

## Litmus ❌

Incorrect address prefix.
Should: 131
Actual: 31

caused by a bug in the script, fix it:
https://github.com/nova-wallet/nova-utils/pull/206

<details>
  <summary>Litmus</summary>
  
Screenshot 2022-02-25 at 07 24 22
Screenshot 2022-02-25 at 07 24 37
Screenshot 2022-02-25 at 07 24 48
Screenshot 2022-02-25 at 07 24 58
Screenshot 2022-02-25 at 07 25 36

</details>

## Crab Parachain ✅

<details>
  <summary>Crab</summary>
 
Screenshot 2022-02-25 at 07 28 29
Screenshot 2022-02-25 at 07 28 55
Screenshot 2022-02-25 at 07 29 03
Screenshot 2022-02-25 at 07 29 11
Screenshot 2022-02-25 at 07 29 31
Screenshot 2022-02-25 at 07 29 43

</details>

File: chains/v2/chains_dev.json [hunk 2]
```diff
@@ -1950,5 +1950,87 @@
...
+                "name": "Litenry node"
+            }
+        ],
+        "types": {
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/litmus.json",
+            "overridesCommon": true
+        },
+        "color": "linear-gradient(315deg, #4A22E1 0%, #7657EF 100%)",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Litmus.svg",
+        "addressPrefix": 31
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:2008]: should be 131, see comment above

Comment (leohar) [reply chains/v2/chains_dev.json]: fixed

**Review by stepanLav — APPROVED** [review]

**Pull request merged**